server/item/music_disc.go

```diff
@@ -0,0 +1,31 @@
+package item
+
+import (
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/df-mc/dragonfly/server/world/sound"
+)
+
+// MusicDisc is an item that can be played in jukeboxes.
+type MusicDisc struct {
+	// DiscType is the disc type of the music disc.
+	DiscType sound.DiscType
+}
+
+// MaxCount always returns 1.
+func (MusicDisc) MaxCount() int {
+	return 1
+}
+
+// AllMusicDiscs returns all 15 music disc items.
+func AllMusicDiscs() []world.Item {
```

Comment:
Can probably just remove this - I know Dyes did something similar but it's been changed in PR since it wasn't consistent with other areas of the codebase.

Reply:
I'd instead iterate through each possible disc type in registration and set DiscType accordingly.